Formula/abyss.rb

```diff
@@ -0,0 +1,51 @@
+class Abyss < Formula
+  desc "Genome sequence assembler for short reads"
+  homepage "http://www.bcgsc.ca/platform/bioinfo/software/abyss"
+  url "https://github.com/bcgsc/abyss/releases/download/2.0.2/abyss-2.0.2.tar.gz"
+  sha256 "d87b76edeac3a6fb48f24a1d63f243d8278a324c9a5eb29027b640f7089422df"
+
+  head do
+    url "https://github.com/bcgsc/abyss.git"
+
+    depends_on "autoconf" => :build
+    depends_on "automake" => :build
+    depends_on "multimarkdown" => :build
+  end
+
+  option :cxx11
+  option "with-maxk=", "Set the maximum k-mer length to N [default is 128]"
+  option "without-test", "Skip build-time tests (not recommended)"
```

Comment:
no running `make check` or `make test` in homebrew/core unless it's security software